src/com/facebook/buck/ide/intellij/lang/android/AndroidResourceModuleRule.java

```diff
@@ -30,11 +30,14 @@
 import com.facebook.buck.rules.TargetNode;
 import com.google.common.collect.ImmutableSet;
 import java.nio.file.Path;
+import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
 
 public class AndroidResourceModuleRule extends AndroidModuleRule<AndroidResourceDescriptionArg> {
```

Comment:
We need a preconditions check to ensure there is only one resource rule labelled `preferred_res_folder` per `BUCK` file/iml

Reply:
Can you also add a test that we throw an error in this case please?